build_library/grub_install.sh

```diff
@@ -140,6 +140,10 @@ info "Generating ${GRUB_DIR}/load.cfg"
 ESP_FSID=$(sudo grub-probe -t fs_uuid -d "${LOOP_DEV}p1")
 sudo_clobber "${ESP_DIR}/${GRUB_DIR}/load.cfg" <<EOF
 search.fs_uuid ${ESP_FSID} root \$root
```

Comment:
is there another option to determine if we've booted from iso?

i suggested doing a search.file /coreos/cpio.gz, but @mjg59 mentioned that wouldn't work well if someone's put cpio.gz on disk somewhere....